spec/requests/admin/configs_spec.rb

```diff
@@ -347,7 +347,7 @@
         end
 
         it "updates logo_png" do
-          expected_default_image_url = URL.local_image("icon.png")
+          expected_default_image_url = SiteConfig.get_field(:logo_png).dig(:default)
```

Comment:
How do you feel about adding this as a method to `SiteConfig` itself?

```ruby
def self.get_default(field)
  get_field(field).dig(:default).call
end
```

Then we could just use `SiteConfig.get_default(:logo_png)`, etc. here in the spec.

Reply:
Makes sense @citizen428 looks more neater. Thank you!